includes/Modules/Analytics_4/Web_Tag.php

```diff
@@ -42,8 +55,7 @@ public function register() {
 	 */
 	protected function enqueue_gtag_script() {
 		if ( did_action( 'googlesitekit_analytics_init_tag' ) ) {
-			// If the gtag script is already registered in the Analytics module, then we need to add G-<MEASUREMENT_ID> configuration only.
```

Comment:
@Hazlitte this comment is still needed. Let's add it back and remove `G-` from it only.